### ISSUE TITLE
Rule proposal: No moot `./` in `new URL()`

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/eslint-plugin-unicorn#1614]
### Description

`new URL()` accepts relative URLs in the form `new URL('foo', url)` and `new URL('./foo', url)`. I would like to enforce the former, as adding the `./` is moot.

### Fail

```js
new URL('./foo', 'https://sindresorhus.com');
```

### Pass

```js
new URL('foo', 'https://sindresorhus.com');
```

**Comment by fisker** [comment]
Personally, I prefer the opposite, it's more clear to me it's a relative path. I even use `./foo.png`, `./foo.css` in html.

**Comment by sindresorhus** [comment]
I guess the rule could enforce either style through an option.

Name suggestions? `url-constructor-relative-url-style`?

**Comment by fisker** [comment]
How about just `relative-url-style`?

**Comment by sindresorhus** [comment]
Sure

**Comment by sindresorhus** [comment]
This is now accepted.
